### ISSUE TITLE
EXCEPTION: No module named EVE_Gnosis.formulas.formulas

### ISSUE THREAD

**Issue by Tetractys** [Pyfa-fit/Pyfa-fit#79]
Hi!
carefully isolated the actual ~/.pyfa moved into ~/.pyfa.safe;
downloaded with git clone https://github.com/Pyfa-fit/Pyfa-fit.git;

this is the first start:

```
OS version: 	Linux-4.10.0-20-lowlatency-x86_64-with-LinuxMint-18.1-serena
Python: 	2.7.12
wxPython: 	3.0.2.0
SQLAlchemy: 	1.0.11
Logbook: 	0.11.4-dev
pyfa version: 2017.04.26 (live) - YC119.3 1.0
pyfa root: /home/davide/bin/pyfa.fit/Pyfa-fit
save path: /home/davide/.pyfa
fs encoding: UTF-8

EXCEPTION: No module named EVE_Gnosis.formulas.formulas

  File "./pyfa.py", line 379, in <module>
    import eos.db
  File "/home/davide/bin/pyfa.fit/Pyfa-fit/eos/db/__init__.py", line 71, in <module>
    from eos.db.saveddata import booster, cargo, character, crest, damagePattern, databaseRepair, drone, fighter, fit, implant, implantSet, loadDefaultDatabaseValues, \
  File "/home/davide/bin/pyfa.fit/Pyfa-fit/eos/db/saveddata/fighter.py", line 27, in <module>
    from eos.saveddata.fit import Fit
  File "/home/davide/bin/pyfa.fit/Pyfa-fit/eos/saveddata/fit.py", line 27, in <module>
    from eos.gnosis import GnosisFormulas, GnosisSimulation
  File "/home/davide/bin/pyfa.fit/Pyfa-fit/eos/gnosis.py", line 5, in <module>
    from EVE_Gnosis.formulas.formulas import Formulas


```
What did I wrong? or, what I have to add to libraries?

**Comment by Tetractys** [comment]
Update: solved. installed EVE_Gnosis-2016.11.28.post2-py2.py3-none-any.whl via pip.
Other errors as esipy will be installed in a similar mode.
Update(2): Even EsiPy has been installed via pip.

Consider, please, to include these two in the future package.
Issue closed.

**Comment by Tetractys** [comment]
Issue closed as per entry #2.
